src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -99,7 +99,7 @@ public Task<string> StartNewAsync(string orchestratorFunctionName, object input)
         /// <param name="eventData">The JSON-serializeable data associated with the event.</param>
         /// <returns>A task that completes when the event notification message has been enqueued.</returns>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1030:UseEventsWhereAppropriate")]
-        public async Task RaiseEventAsync(string instanceId, string eventName, object eventData)
+        public virtual async Task RaiseEventAsync(string instanceId, string eventName, object eventData)
```

Comment:
Let's keep the implementation of `RaiseEventAsync` here but create an abstract method in a base class.